components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -192,6 +192,124 @@ ConversationDriver::ConversationDriver(
 
 ConversationDriver::~ConversationDriver() = default;
 
+std::vector<mojom::ActionGroupPtr> ConversationDriver::GetActionMenuList() {
```

Comment:
Since this is self-contained, let's move this to something like `constants.h`

Reply:
moved the list to `constants.h`